tests/callbacks/test_early_stopper.py

```diff
@@ -1,55 +1,14 @@
-from typing import List, Optional, Sequence
+from typing import List
 
 import pytest
-from torch import tensor
 from torch.utils.data import DataLoader
 
 from composer import Trainer
 from composer.callbacks.early_stopper import EarlyStopper
-from composer.core import State
-from composer.core.callback import Callback
 from composer.core.time import Time, TimeUnit
-from composer.loggers import Logger
-from composer.trainer.devices.device import Device
 from composer.trainer.devices.device_cpu import DeviceCPU
 from composer.trainer.devices.device_gpu import DeviceGPU
-from tests.common import RandomClassificationDataset, SimpleModel, device
-
-
-class TestMetricSetter(Callback):
```

Comment:
I dont think this belongs in `common`, only really used for two specific tests. The threshold for putting in `common` should be high.

Reply:
OK; will move to `tests/metrics`

Reply:
Moved to `tests/metrics`